test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/scaling/SmartRouterLoadBalancing.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.cloud.integrationtests.scaling;
+
+import java.util.List;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.kie.cloud.api.DeploymentScenarioBuilderFactory;
+import org.kie.cloud.api.deployment.Instance;
+import org.kie.cloud.api.scenario.WorkbenchRuntimeSmartRouterKieServerDatabaseScenario;
+import org.kie.cloud.common.provider.KieServerClientProvider;
+import org.kie.cloud.common.provider.KieServerControllerClientProvider;
+import org.kie.cloud.integrationtests.AbstractCloudIntegrationTest;
+import org.kie.cloud.maven.MavenDeployer;
+import org.kie.cloud.maven.constants.MavenConstants;
+import org.kie.server.api.model.KieContainerStatus;
+import org.kie.server.api.model.KieServerInfo;
+import org.kie.server.api.model.ServiceResponse;
+import org.kie.server.api.model.definition.ProcessDefinition;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.client.ProcessServicesClient;
+import org.kie.server.client.QueryServicesClient;
+import org.kie.server.integrationtests.controller.client.KieServerMgmtControllerClient;
+
+public class SmartRouterLoadBalancing extends AbstractCloudIntegrationTest<WorkbenchRuntimeSmartRouterKieServerDatabaseScenario> {
+
+    private KieServerMgmtControllerClient kieControllerClient;
+    private KieServicesClient kieServerClient;
+    private KieServicesClient kieServerClientRouter;
+
+    private static final int PROCESS_NUMBER = 100;
```

Comment:
I would say that 10 should be enough, no need for 100. @jiripetrlik What do you think about that?

Reply:
In my opinion running 100 or processes is relatively faster than the time it takes to deploy the scenario. On the other hand giving jBPM more to crunch might potentially better find some instabilities/race condition. So I think 100 is better in this case.

Reply:
Right, good point.

Reply:
Good point @jhrcek . I will keep number of processes and check that at least one quarter was send to each execution server.